File: code_templates/actions/php/addCodeCuePoint.php
$cuePoint = new KalturaCodeCuePoint;
$cuePoint->code = $_POST["code"];
$cuePoint->description = $_POST["description"];
$cuePoint->startTime = $_POST["startTime"];
$cuePoint->entryId = <%- Lucy.code.variable('answers.entryIdEqual') %>;
$result = $client->cuePoint->add($cuePoint);
<%- Lucy.returnCode('$result') %>

File: code_templates/actions/ruby/addCodeCuePoint.rb
cuePoint = KalturaCodeCuePoint.new();
cuePoint.code = params[:code];
cuePoint.description = params[:description]
cuePoint.start_time = params[:startTime];
cuePoint.entry_id = <%- Lucy.code.variable('answers.entryIdEqual') %>;
result = @@client.cue_point_service.add(cuePoint);
<%- Lucy.returnCode('result') %>
